src/docs/Components/Pagination/code.mdx

```diff
@@ -56,13 +59,14 @@ order: 2
       <use xlink:href="#iconRightL" />
     </svg>
   </a>
-</div>
+</nav>
 ```
 
 <Preview path="pagination-light" />
 
 ## Accessibility and semantic
```

Comment:
You can remove this part now that you have a tab dedicated

Reply:
Fixed